akka-docs/src/main/paradox/typed/persistence-active-active.md

```diff
@@ -0,0 +1,191 @@
+
+This restriction means that in the event of network partitions, and for a short time during rolling re-deploys, `EventSourcedBehaviors`s are unavailable.
+
+
+Active-active event sourcing enables running multiple replicas of each entity. 
+There is automatic replication of every event persisted to all replicas.
+
+For instance, a replica can be run per:
+
+* Data Center 
+* Availability zone or rack
+
+The motivations are:
+
+* Redundancy to tolerate failures in one location and still be operational
+* Serve requests from a location near the user to provide better responsiveness
+* Balance the load over many servers
+
+However, the event handler must be able to **handle concurrent events** as when active-active is enabled
+there is no longer the single writer principal as there is with a normal `EventSourcedBehavior`.
+
+## Relaxing the single writer principal for availability
+
+Taking the example of using active-active to run a replica per data center.
+
+When there is no network partitions and no concurrent writes the events stored by an `EventSourcedBehavior` at one replica can be replicated and consumed by another (corresponding) replica in another data center without any concerns. Such replicated events can simply be applied to the local state.
+
+![images/replicated-events1.png](images/replicated-events1.png)
+
+The interesting part begins when there are concurrent writes by `EventSourcedBehavior`replicas. That is more likely to happen when there is a network partition, but it can also happen when there are no network issues. They simply write at the "same time" before the events from the other side have been replicated and consumed.
+
+![images/replicated-events2.png](images/replicated-events2.png)
+
+The event handler logic for applying events to the state of the entity must be aware of that such concurrent updates can occur and it must be modeled to handle such conflicts. This means that it should typically have the same characteristics as a Conflict Free Replicated Data Type (CRDT). With a CRDT there are by definition no conflicts and the events can just be applied. The library provides some general purpose CRDTs, but the logic of how to apply events can also be defined by an application specific function.
+
+For example, sometimes it's enough to use application specific timestamps to decide which update should win.
+
+To assist in implementing the event handler active-active detects these conflicts.
+
+## API
+
+The same API as regular `EventSourcedBehavior`s is used to define the logic. To enable an entity for active-active
+replication use the factory methods on @api[ActiveActiveEventSourcing]. 
```

Comment:
`ActiveActiveEventSourcing` sounds weird, no? I thought we were leaning toward `ReplicatedEventSourcing`. Shall we reconsider it?

Reply:
Let's leave name discussion to another PR